cirq-google/cirq_google/calibration/xeb_wrapper.py

```diff
@@ -0,0 +1,102 @@
+# Copyright 2021 The Cirq Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import contextlib
+import multiprocessing
+import multiprocessing.pool
+from typing import Optional, Union, Iterator
+
+import cirq
+import cirq.experiments.random_quantum_circuit_generation as rqcg
+import cirq.experiments.xeb_fitting as xebf
+import cirq.experiments.xeb_sampling as xebsamp
+from cirq_google.calibration.phased_fsim import (
+    PhasedFSimCalibrationResult,
+    PhasedFSimCharacterization,
+    LocalXEBPhasedFSimCalibrationRequest,
+    LocalXEBPhasedFSimCalibrationOptions,
+)
+
+
+@contextlib.contextmanager
+def _maybe_multiprocessing_pool(
+    n_processes: Optional[int] = None,
+) -> Iterator[Union['multiprocessing.pool.Pool', None]]:
+    if n_processes == 1:
+        yield None
+        return
```

Comment:
This is instead of else: ? Mixing yield and return looks a little bit strange at first sight.